fmdtools/analyze/graph.py

```diff
@@ -1032,96 +1159,182 @@ def data_average(data):
         list_average.append(sum(x[i] for x in data)/len(data))
     return list_average
 
-def data_error(data,average):
-    """Calculates error for each column in data"""
+def data_error(data, average):
+    """
+    Calculates error for each column in data
+
+    Parameters
+    ----------
+    data
+    average
+
+    Returns
+    -------
+
+    """
     q1 = []
     q3 = []
-    for i in range(0,len(data[0])):
+    for i in range(0, len(data[0])):
         current_array = np.array([float(x[i]) for x in data])
-        q1.append(np.percentile(current_array,25))
-        q3.append(np.percentile(current_array,75))
-    lower_error = [x - y for x, y in zip(average,q1)]
-    upper_error = [x - y for x, y in zip(q3,average)]
+        q1.append(np.percentile(current_array, 25))
+        q3.append(np.percentile(current_array, 75))
+    lower_error = [x - y for x, y in zip(average, q1)]
+    upper_error = [x - y for x, y in zip(q3, average)]
     return lower_error, upper_error
+
+
 def gv_import_check():
-    """Checks if graphviz is installed on the system before plotting."""
+    """
+    Checks if graphviz is installed on the system before plotting.
+    Returns
+    -------
+
+    """
     try:
         from graphviz import Digraph, Graph
     except ImportError as error:
         print(error.__class__.__name__ + ": " + error.message)
         raise Exception("GraphViz not installed. Please see:\n https://pypi.org/project/graphviz/ \n https://www.graphviz.org/download/")
     return Digraph, Graph
 
-class GraphInteractor: 
+
+class GraphInteractor:
     """A simple interactive graph for consistent node placement, etc--used in set_pos to set node positions"""
     showverts = True
     epsilon = 0.2  # max pixel distance to count as a vertex hit
+
     def __init__(self, g_obj, **kwargs):
-        self.t=0
-        self.fig, (self.bax, self.ax) = plt.subplots(2, gridspec_kw={'height_ratios': [1,10]})
-        self.kwargs=kwargs
-        self.g_obj=g_obj
+        """
+
+        Parameters
+        ----------
+        g_obj
+        kwargs
```

Comment:
g_obj : Graph 
    Graph object to plot interactively
kwargs : dict
    kwargs for Graph.draw

Reply:
Done